### PR TITLE
Include Timescale as adopter

### PR DESCRIPTION
**Pull request by adolsalamanca**
Just saw @jpkrohling [LinkedIn post](https://www.linkedin.com/posts/jpkroehling_opentelemetry-graduation-by-austinlparker-activity-7172873030925332480-FvMW) an decided to include us in the list,
as we have been using OpenTelemetry for a while already.

Thanks!

### CODE REVIEW

**Comment by theletterf** [comment]
Hi! Could you sign the CLA?

**Comment by adolsalamanca** [comment]
Hi @theletterf , I have tried to sign it already and it's awaiting for my employer authorization.
Thanks

**Comment by svrnm** [comment]
> Thanks @jpkrohling , I believe it's in the appropriate section. Timescale is now just focused on offering the best cloud timeseries/vector/analytics database, built on PostgreSQL.

Your blog post indicates otherwise, it's not about how timescale uses OpenTelemetry internally to monitor their own infrastructure. Especally the last sentence indicates that this is better suited in the vendors list: "If you're looking for a time-series database to store your metrics, check out Timescale."

**Comment by adolsalamanca** [comment]
> > Thanks @jpkrohling , I believe it's in the appropriate section. Timescale is now just focused on offering the best cloud timeseries/vector/analytics database, built on PostgreSQL.
> 
> Your blog post indicates otherwise, it's not about how timescale uses OpenTelemetry internally to monitor their own infrastructure. Especally the last sentence indicates that this is better suited in the vendors list: "If you're looking for a time-series database to store your metrics, check out Timescale."

Thanks for your response @svrnm , that's accurate. 
Maybe we should not include that blog post, as it's from times when we were actively promoting [Promscale](https://github.com/timescale/promscale), which has been already discontinued.

**Review by svrnm — CHANGES_REQUESTED** [review]

File: data/ecosystem/adopters.yaml [hunk 1]
```diff
@@ -120,3 +120,10 @@
   components: []
   reference: ''
   contact: ''
+- name: Timescale
+  url: https://www.timescale.com/
+  components: [Collector, Go]
+  reference: blog/a-deep-dive-into-open-telemetry-metrics/
```
[the remark this answers is not in the document]
Comment (adolsalamanca) [reply data/ecosystem/adopters.yaml]: Thanks, just fixed it, but then a few other needs an update too.

[the remark this answers is not in the document]
Comment (svrnm) [reply data/ecosystem/adopters.yaml]: ebay, farfetch and uplight blogs are local at opentelemetry.io

[the remark this answers is not in the document]
Comment (adolsalamanca) [reply data/ecosystem/adopters.yaml]: Oh, I had no idea.
thanks

[the remark this answers is not in the document]
Comment (svrnm) [reply data/ecosystem/adopters.yaml]: > Please replace this with a blog post that shows how timescale uses OpenTelemetry themselves

this is still open.

[the remark this answers is not in the document]
Comment (adolsalamanca) [reply data/ecosystem/adopters.yaml]: Sorry, didn't read your  comment correctly.
I'm removing the post altogether as there's no such a blog post, we can add it later when the the time comes.

thanks

[the remark this answers is not in the document]
Comment (svrnm) [reply data/ecosystem/adopters.yaml]: @adolsalamanca to me it is still not clear how Timescale "adopts" OpenTelemetry, are you using it internally to monitor your own systems, or are you providing services to customers that make use of OpenTelemetry or can be used in combination with OpenTelemetry. The first instance is "adoption", the other 2 instances are either a "vendor" (if you ingest OpenTelemetry into your systems for your customers") or integration (if your services emit OpenTelemetry to observability backends)

**Comment by svrnm** [comment]
@adolsalamanca any updates?

**Comment by cartermp** [comment]
Closing old PR

**Pull request closed without merging**